apps/src/aiTutor/accessControlsApi.ts

```diff
@@ -0,0 +1,63 @@
+import {getAuthenticityToken} from '@cdo/apps/util/AuthenticityTokenStore';
+import MetricsReporter from '@cdo/apps/lib/metrics/MetricsReporter';
+import {MetricEvent} from '@cdo/apps/lib/metrics/events';
+import {StudentServerData} from './types';
+
+const formatServerData = (student: StudentServerData) => ({
+  id: student.id,
+  name: student.name,
+  aiTutorAccessDenied: student.ai_tutor_access_denied,
+});
+
+export const handleUpdateAITutorAccess = async (
+  userId: number,
+  newAccess: boolean
+) => {
+  try {
+    const response = await fetch(`/api/v1/users/${userId}/ai_tutor_access`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-CSRF-Token': await getAuthenticityToken(),
+      },
+      body: JSON.stringify({ai_tutor_access: newAccess}),
+    });
+    if (!response.ok) {
+      throw new Error(`HTTP error! Status: ${response.status}`);
+    }
+  } catch (error) {
+    MetricsReporter.logError({
+      event: MetricEvent.AI_TUTOR_UPDATE_USER_ACCESS_FAIL,
+      errorMessage: JSON.stringify(error),
+    });
```

Comment:
Worth pointing out the new metric event in the PR summary, good breadcrumb for future metric searchers.